onnx/numpy_helper.py

```diff
@@ -231,6 +228,14 @@ def to_array(tensor: onnx.TensorProto, base_dir: str = "") -> np.ndarray:  # noq
     return np.asarray(data, dtype=storage_np_dtype).astype(np_dtype).reshape(dims)
 
 
+def tobytes_little_endian(array: np.ndarray) -> bytes:
```

Comment:
Could you add a docstring for it as it is a public method? Be sure to also add an `.. versionadded:: 1.20` line. (https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded)

Reply:
This is looking good. Thanks.